src/MRich_geo.cpp

```diff
@@ -1,465 +0,0 @@
-// SPDX-License-Identifier: LGPL-3.0-or-later
```

Comment:
Why remove Mrich?

Reply:
I mean why in this PR.

Reply:
I don't know. I am not sure how that happened. It's now back